src/Http/samples/MinimalSample/Program.cs

```diff
@@ -9,6 +10,8 @@
 var builder = WebApplication.CreateBuilder(args);
 var app = builder.Build();
 
+app.Logger.LogInformation($"Current process ID: {Process.GetCurrentProcess().Id}");
```

Comment:
It should be `Environment.ProcessId` as per https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1837